roles/openshift_examples/tasks/main.yml

```diff
@@ -49,18 +49,26 @@
 ######################################################################
 # Begin image streams
 
-- name: Create imagestream import secret
+- name: Create imagestream import secret for oreg_url
   command: >
     {{ openshift_client_binary }} create secret docker-registry imagestreamsecret --docker-server={{ registry_host }} --docker-username={{ oreg_auth_user }} --docker-email=openshift@openshift.com --docker-password={{ oreg_auth_password }} --config={{ openshift.common.config_base }}/master/admin.kubeconfig -n openshift
   when:
     - openshift_examples_load_rhel | bool
     - oreg_auth_password is defined
-  with_items:
-    - "{{ rhel_image_streams }}"
   register: oex_imagestream_import_secret
   failed_when: "'already exists' not in oex_imagestream_import_secret.stderr and oex_imagestream_import_secret.rc != 0"
   changed_when: false
 
+- name: Create imagestream import secrets for any additional registries
+  command: >
+      {{ openshift_client_binary }} create secret docker-registry imagestreamsecret --docker-server={{ item.host }} --docker-username={{ item.user }} --docker-email=openshift@openshift.com --docker-password={{ item.password }} --config={{ openshift.common.config_base }}/master/admin.kubeconfig -n openshift
```

Comment:
Break up this line for readability.  No sense in using '>' if we're not going to utilize multiple lines.

Reply:
ok